test.sh

```diff
@@ -23,19 +25,33 @@ function host {
 }
 
 docker rm -fv func-postgres-test || echo No prev test db container
-docker run --name func-postgres-test -e "POSTGRES_DB=funcs" -e "POSTGRES_PASSWORD=root" -p 5432:5432 -d postgres
 docker rm -fv func-mysql-test || echo No prev mysql test db container
-docker run --name func-mysql-test -p 3306:3306 -e MYSQL_DATABASE=funcs -e MYSQL_ROOT_PASSWORD=root -d mysql
 docker rm -fv func-minio-test || echo No prev minio test container
-docker run -d -p 9000:9000 --name func-minio-test -e "MINIO_ACCESS_KEY=admin" -e "MINIO_SECRET_KEY=password" minio/minio server /data
 
 # build test image locally first
-(cd images/fn-test-utils && ./build.sh)
+(cd images/fn-test-utils && ./build.sh) &
+PID1=$!
 
 # pull all images used in tests so that tests don't time out and fail spuriously
-docker pull fnproject/sleeper
-docker pull fnproject/error
-docker pull fnproject/hello
+( docker pull fnproject/sleeper && docker pull fnproject/error && docker pull fnproject/hello) &
+PID2=$!
+
+# pull db images
+( docker pull postgres && docker pull mysql && docker pull minio/minio) &
+PID3=$!
+
+# pull swagger image
+(docker pull quay.io/goswagger/swagger ) &
```

Comment:
see my comments at related issue, this image huge, why not just download binary for releases?